src/defaultContent/pages/apis/partials/api-listing.hbs

```diff
@@ -96,41 +97,48 @@
+                            alt="Subscribed" class="subscription-icon" />{{/if}}
+                        </div>
 
-                      {{#if (eq ../subscriptionPolicies.length 1)}}
-                      <div class="select-item {{#if subscribed}}disabled{{/if}}" role="button" data-value="{{id}}"
-                        onclick="document.getElementById('selectedAppId-{{../apiID}}').value='{{id}}'; subscribe('{{../../orgID}}', '{{id}}', '{{../apiID}}', '{{../apiReferenceID}}', 
-                        '{{lookup (lookup ../subscriptionPolicyDetails 0) 'policyID'}}', 
-                        '{{lookup (lookup ../subscriptionPolicyDetails 0) 'policyName'}}' )">
-                        <span>{{name}}</span>
-                        {{#if subscribed}}<img
-                          src="https://raw.githubusercontent.com/wso2/docs-bijira/refs/heads/main/en/devportal-theming/success-rounded.svg"
-                          alt="Subscribed" class="subscription-icon" />{{/if}}
+                        {{/each}}
                       </div>
-                      {{else}}
-                      <div class="select-item {{#if subscribed}}disabled{{/if}}" role="button" data-value="{{id}}"
-                        onclick="document.getElementById('selectedAppId-{{../apiID}}').value='{{id}}'; loadModal('planModal-{{../apiID}}')">
-                        <span>{{name}}</span>
-                        {{#if subscribed}}<img
-                          src="https://raw.githubusercontent.com/wso2/docs-bijira/refs/heads/main/en/devportal-theming/success-rounded.svg"
-                          alt="Subscribed" class="subscription-icon" />{{/if}}
+                      <div class="create-app-container" style="display: none;">
+                        <div class="create-app-option" role="button">
+                          Create application "<span class="search-term"></span>" and Subscribe
+                        </div>
                       </div>
-                      {{/if}}
-
-                      {{/each}}
                     </div>
                   </div>
                 </div>
-                {{/if}}
+
+                {{#if (eq subscriptionPolicies.length 1)}}
+                 <a type="button" class="common-btn-primary"
+                   onclick="showSubscribeButtonLoading(this); subscribe('{{../orgID}}', '', '{{apiID}}', '{{apiReferenceID}}', '{{lookup (lookup subscriptionPolicyDetails 0) "policyID"}}', '{{lookup (lookup subscriptionPolicyDetails 0) "policyName"}}')"
+                   disabled>Subscribe</a>
+                 {{else}}
+                 <a type="button" class="common-btn-primary" onclick="showSubscribeButtonLoading(this); loadModal('planModal-{{apiID}}')" disabled>Subscribe</a>
+                 {{/if}}
+
                 {{/if}}
               </div>
+              {{/if}}
             </div>
           </div>
         </div>
       </div>
       {{/apiMetadata}}
     </div>
   </div>
-</section>
+</section>
```

Comment:
@Piumal1999 shall we add a newline at the end?

Reply:
Fixed with https://github.com/wso2/api-developer-portal-core/pull/567